src/client/content_cao.cpp

```diff
@@ -634,8 +636,12 @@ void GenericCAO::addToScene(ITextureSource *tsrc, scene::ISceneManager *smgr)
 		u32 shader_id = shader_source->getShader("object_shader", material_type, NDT_NORMAL);
 		m_material_type = shader_source->getShaderInfo(shader_id).material;
 	} else {
-		m_material_type = (m_prop.use_texture_alpha) ?
-			video::EMT_TRANSPARENT_ALPHA_CHANNEL : video::EMT_TRANSPARENT_ALPHA_CHANNEL_REF;
+		if (m_prop.use_texture_alpha) {
+			m_material_type = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
+			m_material_type_param = 1.0f / 256.f; // minimal alpha for texture rendering
```

Comment:
Wouldn't 0.0f have the same effect?

Reply:
:shrug:

Reply:
Well, is it a `<` or `<=`? :sweat_smile:

Reply:
@appgurueu In L623 I put a note regarding this value. `0.5f` cuts off values below 128, hence this code will cut off at alpha = 0 which is already transparent.